src/analysis/ReachingDefinitions/Ssa/PhiPlacement.h

```diff
@@ -75,17 +75,15 @@ class PhiPlacement
         std::vector<std::unique_ptr<RDNode>> result;
         for (auto& pair : pa) {
```

Comment:
The same (merge)

Reply:
Also, `std::make_unique` is a C++14 feature, right? Can not there be a problem, if we require only C++11?

Reply:
Yes, it is and it would cause the code not to compile. 
If you are referring to this piece of code involving `push_back`, it does not use `make_unique`. That is invocation of constructor of `std::unique_ptr<T>` which should be present since C++11.
Also, I've used `make_unique` from llvm in a few places.